web/src/main.ts

```diff
@@ -1,10 +1,17 @@
 import 'focus-visible';
 import * as React from 'react';
 import { render } from 'react-dom';
+import TagManager from 'react-gtm-module';
 import './components/index.css';
 
 declare var require: any;
 
+const tagManagerArgs = {
+  gtmId: process.env.GTMID,
```

Comment:
I'm curious how we pass env variables on the front end. I've not seen examples of where we use env variables